include/AutomatableModel.h

```diff
@@ -435,6 +435,14 @@ template <typename T> class LMMS_EXPORT TypedAutomatableModel : public Automatab
 		return AutomatableModel::value<T>( frameOffset );
 	}
 
+	T valueAt(size_t index)
```

Comment:
I am seriously thinking of removing this function and revert back to the state where everything was working. I did try something now. I'll see anyway.